### DISCUSSION TITLE
Ability to Import Open Tablet Driver config file

### DISCUSSION THREAD

**Discussion by daveycombs** [ppy/osu#25134] [Ideas]
Allow the ability to import Open Tablet driver configurations into Lazer
Decimal precision to edit area

**Comment by peppy** [comment]
We will be adding manual entry. but likely not a configuration file import.

**Comment by slopslug** [comment]
You can change the entries' in the input configuration file if that's what you want.

In the configuration folder for lazer, (on Windows it should be `%APPDATA%\osu`) there exists a file named `input.json`. This is where your tablet area configuration is stored. This is where you should be changing the values. 
It's not the most intuitive, but it should do for now until there is (hopefully) a better way.
